tables/en-ueb-g2.ctb

```diff
@@ -988,7 +997,7 @@ match - en %1*!%[_1] 26
 
 nofor always en 26
 
-midendword eness 15-56-234 closeness
+match %a eness - 15-56-234b closeness
```

Comment:
Why is `match` better than `midendword`? As far as I can see you are just replacing a the `midendword` opcode with the exact same behaviour implemented using `match`. Generally I like more specific opcodes better because as a reader you grasp the meaning much faster than having to understand the meaning of the more general `match` opcode.

Also why is there now a virtual dot `b` at the end of the braille dots?

Reply:
The virtual dots were introduced in #1491 to deal with the fact that final letter groupsigns (e.g. -ful, -less, -ness) shouldn't be used if they follow capital-related indicators (as in `dAnce` or `getFullName`).  There is a second pass that removes them and/or substitutes the expanded letters where needed if capital start/end indicators are detected.

Reply:
Whether some of our more specific opcodes are much easier to understand than the general-purpose `match` is debatable.